awsiot-optee/aws-iot-setup.py

```diff
@@ -160,6 +160,10 @@ def register_ca(ca_cert: str, ca_key: str, reg_conf: str):
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser()
+    parser.add_argument('-r', '--role-name', default=iot_role_name,
+        help='AWS IOT role name')
```

Comment:
`help='AWS IOT role name. Default=%(default)s'` to make a better help message.

You can use this `%(default)s` trick through all your new help options to improve this.

Also - all your strings are using single quotes. This file is using double-quotes throughout. Let's try and keep the style consistent.